Assets/MRTK/SDK/Features/Input/Handlers/DefaultTransformSmoothingLogic.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// A default implementation of TransformSmoother for smoothing transforms.
+    /// </summary>
+    public class DefaultTransformSmoothingLogic : ITransformSmoothingLogic
```

Comment:
These new files seem a little out of place in the Handlers folder. Especially since they're in the Utilities namespace, could they be moved to `MRTK/SDK/Features/Utilities`?

Reply:
I can do this so I'm not blocking the PR

Reply:
Updated and approved!